Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp

```diff
@@ -12,7 +12,7 @@ DivEFunctor::operator()(amrex::MultiFab& mf_dst, const int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
     // Guard cell is set to 1 for generality. However, for a cell-centered
-    // output Multifab, mf_avg, the guard-cell data is not needed especially considering
+    // output Multifab, m_mf_output, the guard-cell data is not needed especially considering
```

Comment:
Same comments, which also holds in other files below.

Reply:
Made this modification to mf_dst as well.